src/bindings/js/node/package.json

```diff
@@ -22,5 +34,16 @@
   },
   "engines": {
     "node": ">=18.16.0"
```

Comment:
Consider to synchronize `engines.node` and `devDependencies.@types/node` version.